scrunch/datasets.py

```diff
@@ -371,12 +383,23 @@ def __init__(self, obj, order, parent=None):
         # Load all the elements.
         for element in obj[self.name]:
             if isinstance(element, six.string_types):
-                _id = element.split('/')[-2]
-                var = self.order.vars.get(_id)
-                if var:
-                    self.elements[var['alias']] = var
+                if 'datasets' not in element or 'variables' in element:
```

Comment:
Sure thing. `ds.variables.orders.hier` returns relative URL's to its variables while `project.datasets.order` returns absolute URL's. So I saw these 3 possibilities here:
- relative URLs to variables
- absolute URLs to variables (might change in the future)
- absolute URLs to datasets
- (touché, I missed relative URLs to datasets which could also change in the future)
I will take a look at pycrunch to handle this.